docs/examples/google-cloud-function.ts

```diff
@@ -0,0 +1,29 @@
+import { Telegraf } from 'telegraf'
+
+const { BOT_TOKEN, PROJECT_ID, FUNCTION_NAME, REGION } = process.env
+
+// eslint-disable-next-line
+const bot = new Telegraf(`${BOT_TOKEN}`, {
+  telegram: { webhookReply: true },
+})
+
+// eslint-disable-next-line
```

Comment:
Why is this needed?
```suggestion
```

Reply:
setWebhook returns a promise and the eslint rules are set to not allow unhandled promises

Reply:
you can explictly mark which rule this is disabling:

```suggestion
// eslint-disable-next-line @typescript-eslint/no-floating-promises
```